components/server/src/server.rs

```diff
@@ -369,7 +369,11 @@ impl<ER: RaftEngine> TiKVServer<ER> {
 
         if tikv_util::panic_mark_file_exists(&self.config.storage.data_dir) {
             fatal!(
-                "panic_mark_file {} exists, there must be something wrong with the db.",
+                "panic_mark_file {} exists, there must be something wrong with the db. \
+                     Do not remove the panic_mark_file and force the TiKV node to restart. \
```

Comment:
The line seems redundant, people may not realize it can be deleted.

Reply:
We have seen several user who upon seeing the error just manually delete the panic mark and force TiKV restart. We should have clear message to prevent them doing so.